cmd/disco/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/google/gnostic/cmd/disco
+
+go 1.12
+
```

Comment:
Should these go.mod files also include the following?
```
replace github.com/google/gnostic => ../..
```
I'm concerned that the gnostic version reference (currently v0.5.7) will be hard to maintain.

Reply:
@timburks Thank you for your reply. Replace cannot be added to go of command line tool type In go.mod, It will cause the installation to fail.
https://github.com/golang/go/issues/44840

Reply:
You can consider modifying the dependent versions in these gomod files after publishing the version

Reply:
Or you can specify a new version when there is no release.
https://github.com/go-kratos/kratos/commit/1117350b9c47f638709edb03456a88f5a05573cd